pom.xml

```diff
@@ -192,6 +192,8 @@
         <dep.flyway.version>10.10.0</dep.flyway.version>
         <dep.google.http.client.version>1.44.1</dep.google.http.client.version>
         <dep.iceberg.version>1.5.0</dep.iceberg.version>
+        <!-- temporary until updated via Airbase -->
+        <dep.jackson.version>2.17.0</dep.jackson.version>
```

Comment:
How does this relate to Nessie update?

Reply:
resolves upper bound version conflict. I'll rebase on the https://github.com/trinodb/trino/pull/21155 which will land before this one.

Reply:
Commit dropped.